### PR TITLE
GDS Admin Ui: fix some issues on status component of dashboard

### PR DESCRIPTION
**Pull request by masskoder**
Ensure that the status component does not raise errors when it receives an empty object or undefined value

### CODE REVIEW

File: web/gds-admin-ui/src/pages/app/dashboard/Status.js [hunk 1]
```diff
@@ -91,30 +93,28 @@ const Status = ({ statuses }) => {
                             <Col xs={4}>
                                 <i className="mdi mdi-progress-question text-warning mt-3 h4"></i>
                                 <h4 className="fw-normal">
-                                    <span>{statusPercents()[STATUS.PENDING_REVIEW] + '%'}</span>
+                                    <span>{statusPercents()[STATUS.PENDING_REVIEW] || 0 + '%'}</span>
```
Comment (elysee15) [inline web/gds-admin-ui/src/pages/app/dashboard/Status.js:96]: Thank you for those fixes

**Review by elysee15 — APPROVED** [review]

File: web/gds-admin-ui/src/pages/app/dashboard/Status.js [hunk 2]
```diff
@@ -91,30 +93,28 @@ const Status = ({ statuses }) => {
...
                                 </h4>
                                 <p className="text-muted mb-0 fs-6">Pending</p>
                             </Col>
 
                             <Col xs={4}>
                                 <i className="mdi mdi-alert-octagram text-danger mt-3 h4"></i>
                                 <h4 className="fw-normal">
-                                    <span>{statusPercents()[STATUS.REJECTED] + '%'}</span>
+                                    <span>{statusPercents()[STATUS.REJECTED] || 0 + '%'}</span>
```
Comment (pdeziel) [inline web/gds-admin-ui/src/pages/app/dashboard/Status.js:104]: Looks like this is unintentionally stripping off the percentages, I'm not a JS expert but do we need to add some parentheses here?

Screen Shot 2022-02-07 at 9 28 07 AM

**Review by pdeziel — CHANGES_REQUESTED** [review]
Thanks for opening this PR! I did encounter an issue with the percentage symbols not rendering (see comment below), so I would like to get that fixed before we merge this in.

File: web/gds-admin-ui/src/pages/app/dashboard/Status.js (hunk 2 above)
Comment (masskoder) [reply web/gds-admin-ui/src/pages/app/dashboard/Status.js]: @pdeziel good catch 👍  thanks. I added the missing percent symbols.

File: web/gds-admin-ui/src/pages/app/dashboard/Status.js [hunk 3]
```diff
@@ -10,51 +10,52 @@ const Status = ({ statuses }) => {
...
                     break;
                 default:
-                    counts[STATUS.PENDING_REVIEW] += status[1]
+                    counts[STATUS.PENDING_REVIEW] += status[1];
                     break;
             }
-            return counts
-        }
+            return counts;
+        };
```
Comment (pdeziel) [inline web/gds-admin-ui/src/pages/app/dashboard/Status.js:28]: Will the linter help us catch these?

File: web/gds-admin-ui/src/pages/app/dashboard/Status.js [hunk 4]
```diff
@@ -91,37 +94,35 @@ const Status = ({ statuses }) => {
                             <Col xs={4}>
                                 <i className="mdi mdi-progress-question text-warning mt-3 h4"></i>
                                 <h4 className="fw-normal">
-                                    <span>{statusPercents()[STATUS.PENDING_REVIEW] + '%'}</span>
+                                    <span>{`${statusPercents()[STATUS.PENDING_REVIEW] || 0} %`}</span>
```
Comment (pdeziel) [inline web/gds-admin-ui/src/pages/app/dashboard/Status.js:97]: Thanks for figuring this out!

**Review by pdeziel — APPROVED** [review]
Thanks for figuring this out @masskoder, everything looks good on my end!

**Comment by bbengfort** [comment]
Closing in favor of #326 so that we can merge a signed commit. I'll carry over @pdeziel and @elysee15's review to that PR.

**Pull request closed without merging**